Creiwry/kitten_project
# Create order items at stripe checkout

### Ticket Link
https://trello.com/c/yIMz78bQ

### User Story
https://trello.com/c/atJp3THP

### What does this PR do?
It adds a method to create order items when a stripe order is passed.

### Where should the reviewer start?
The changes took place in the checkout controller and the order model.

### How should this be manually tested?
Do a checkout of an item with stripe, and check if the item is visible in the user's profile.

### Are there any breaking changes?
no
